### PR TITLE
New drone config

### PR DESCRIPTION
**Pull request by projectweekend**

### CODE REVIEW

**Comment by WorkHorseIndustries** [comment]
Sup with this?

File: .drone.yml [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
-    - go get -t
-    - go build
-    - go test -gocheck.v -coverprofile=coverage.txt -covermode=count
-    - cd fetch && go test -v -coverprofile=coverage.txt -covermode=count
-    - cd ..
-    - cat fetch/coverage.txt >> coverage.txt && rm fetch/coverage.txt
-    - cat coverage.txt
-    - curl -sSL https://cvr.vokal.io/upload | bash
+build:
+    image: registry.vokal.io/go1.5
```
Comment (scottferg) [inline .drone.yml]: Definitely want 1.6 for this

**Comment by foresmac** [comment]
@projectweekend If you're able to contact Andrew, I think he did a lot of work to get this building in a container. He might have some stuff that helps here.

**Comment by scottferg** [comment]
It looks like the build errors are just the result of an ancient build container. I'd start by switching to `golang` and then rebuilding step by step

**Comment by foresmac** [comment]
Yeah, I'm just saying he already did all that and got it building.

**Pull request closed without merging**